api/optimeet/groups/urls.py

```diff
@@ -5,5 +5,8 @@
 
 urlpatterns = [
     path("", views.groups),
-    path("<int:group_id>/", views.group_detail)
+    path("<int:group_id>/", views.group_detail),
+    path('<int:group_id>/users/', views.add_users_to_group, name='add_users_to_group'),
+    path('usergroups/', views.UserGroupListAPIView.as_view(), name='usergroup-list'),#get all usergroup
+    path('usergroups/<str:user_id>/', views.UserGroupListAPIView.as_view(), name='usergroup-byuser'),#get usergroup by user_id
```

Comment:
These routes will end up being `/groups/usergroups/` and `/groups/usergroups/<user_id>/` since they're inside the `groups` app. This route naming is not very standard, so if we're trying to create a route that will get all groups a user is a part of (GET api/users/{id}/groups/), then let's instead create a new `users` app and move the `UserGroups` model and `UserGroupsSerializer` to it. We can then create this new route and import `UserGroupsSerializer` in `groups/views.py`.

If you don't think working on this right now is worth it, we could leave it for later, since we won't be needing to get the groups a user is a part of for the time being (since you can't see the groups you are a part of in the home page rn). We can always discuss it in Discord!

Reply:
okay can we leave it for later?

Reply:
Ok let's comment out these routes for the time being then.